command-line-interface.md

```diff
@@ -0,0 +1,189 @@
+# Operations
+
+A conformant [runtime][] MUST provide an executable (called `funC` in the following examples).
```

Comment:
On Fri, Jul 29, 2016 at 01:05:39PM -0700, Mike Brown wrote:
> As much as it pains me to say it, I can't come up with a good reason
> to disagree with you regarding putting the runtime cli spec in an
> opencontainers/runtime-client-spec repo.

I have no problem with putting it in
opencontainers/command-line-runtime-api or some such (we want to keep
space for http-runtime-api, etc.).  Keeping things in one repository
makes for easier locked-versioning, but I don't see a need to lock the
command line API versioning to the configuration versioning.

Multiple repositories may (or may not, the TOB is unclear on this
point, opencontainers/tob#2) mean separate OCI Projects, and I'm fine
with either a new command-line-runtime-api Project or having both
command-line-runtime-api and runtime-spec be under the runtime-spec
Project (with the distinction being mostly whether the maintainer sets
are identical).

So are we closing this PR, reviving [1], transfering it to
opencontainers/…, and formally picking maintainers?  @philips, should
I work up a TOB proposal for this?

[1]: https://github.com/wking/oci-command-line-api